codespan-reporting/src/diagnostic.rs

```diff
@@ -112,6 +112,8 @@ impl<FileId> Label<FileId> {
 
 /// Represents a diagnostic message that can provide information like errors and
 /// warnings to the user.
+///
+/// The position of a Diagnostic is considered to be the position of the first [`Label`] with a style of [`LabelStyle::primary`].
```

Comment:
Do we need this comment anymore, seeing as we now get the primary label with the smallest start position?

Reply:
That is what I was trying to say: The Position of the first label with a style of primary, so the first primary label. But I see how "first" vs. "smallest start position" is confusing. Maybe like this:

```suggestion
/// The position of a Diagnostic is considered to be the position of the [`Label`] with a style of [`LabelStyle::primary`] that has the smallest start position.
```

Or should it be removed completely? I just thought that it would be good to document this behaviour.

Reply:
Ahh yeah, I'm not yet decided on if we want to talk about presentation stuff here - but on the other hand, that's where people will probably look first. Let's commit your rewording. :+1: